config-templates/azure/azuredisk-csi-driver/1.18.0/custom-parameters.json

```diff
@@ -64,13 +64,13 @@
         "mutable": "true"
     },
     {
-        "description": "The security group name. You can find your security group name in the Azure portal by running the 'az network nsg list' command.",
+        "description": "The name of your security group. You can find your security group details in the Azure portal or by running the `az network nsg list` command.",
         "displayname": "Network Security Group Name",
         "name": "securityGroupName",
         "required": "true"
     },
     {
-        "description": "The name of the virtual network. You can find the name of your virtual network in the Azure portal or by running the 'az network vnet list' command.",
+        "description": "The name of the virtual network. You can find this value in the Azure portal or by running the `az network vnet subnet list` command.",
```

Comment:
Here also the command is not correct.
It will be
`az network vnet list`

Reply:
Same will be applicable in 1.23.0 as well

Reply:
Same will be applicable in 1.4.0 as well

Reply:
Fixed